octree/include/pcl/octree/impl/octree_iterator.hpp

```diff
@@ -258,14 +257,13 @@ OctreeBreadthFirstIterator<OctreeT>::operator++()
 
 //////////////////////////////////////////////////////////////////////////////////////////////
 template <typename OctreeT>
-OctreeFixedDepthIterator<OctreeT>::OctreeFixedDepthIterator()
-: OctreeBreadthFirstIterator<OctreeT>(0u), fixed_depth_(0u)
```

Comment:
Why did you change this line?

Reply:
Accidentally? I don't recall making this change. I might have deleted something by accident and the formatter updated the line.